docs/CHANGELOG.rst

```diff
@@ -38,6 +38,7 @@ Added
 - Add ``background_pairs`` Jinja expressions filter that accepts a list of
   data objects and orders them in a list of pairs (case, background) based on
   the background relation between corresponding samples.
+- Add ERCC table to ``resolwebio/rnaseq`` docker image.
```

Comment:
Add version of docker image: it will be `4.1.0`. Please put exact version also in the commit message.